jsx/Filter.js

```diff
@@ -77,6 +78,14 @@ class Filter extends Component {
     }, []);
   }
 
+  renderActions() {
```

Comment:
renderActions() is never actually called

Reply:
Yeah, looks like a lot of this is due to the face that for a brief period of time the actions were being rendered in the filter instead of the datatable component. I'll clean it up.